src/browser/api/system.js

```diff
@@ -249,6 +250,9 @@ exports.System = {
     getConfig: function() {
         return coreState.getStartManifest();
     },
+    getCrashReporterState: function() {
```

Comment:
mmmmmmm i will if at least one more person wants it... id rather them be consistent and i dont want to change them all